src/reference-data-types.js

```diff
@@ -0,0 +1,43 @@
+module.exports = [
```

Comment:
Do all of these types return `{ name, code, deprecated }`?

Reply:
Yes and no, there may be more properties.

Reply:
For VehicleSupplierType it is
`{
	"Code": "CI",
	"Description": "Choice International - Comfort Inns",
	"Deprecated": "false"
}`